manifests/20-performance-profile.crd.yaml

```
@@ -528,6 +528,12 @@ spec:
                     offlined:
                       description: Offline defines a set of CPUs that will be unused and set offline
                       type: string
+                    ovsDpdk:
```

YAMLlint (1.37.1)

[error] duplication of key "ovsDpdk" in mapping

(key-duplicates)